### ISSUE TITLE
Total number of healthsites?

### ISSUE THREAD

**Issue by LeMeteore** [healthsites/healthsites#920]
Hoping that I did not miss a detail in the documentation, is there a specific parameter to send with the request,  is there a specific request to send so that I can retrieve only the total number of healthsites (per country or per geographical area or per creation date)? Thanks.

**Comment by todrobbins** [comment]
Hi there! Did you already look through the [API documentation](https://github.com/healthsites/healthsites/wiki/API)? I don't think there is a parameter currently that just gives a total count, but I think you could enumerate the total based on the number of results.

**Comment by LeMeteore** [comment]
> Did you already look through the API documentation?

Hi, yes, I did, and indeed, I did not find anything. I asked for being sure.

>  I think you could enumerate the total based on the number of results.

According to the [documentation](https://github.com/healthsites/healthsites/wiki/API#11-getting-all-facilities), I can retrieve healthsites facilities only by chunks of 100 healthsites maximum. Example, the last url returning a result is: `https://healthsites.io/api/v1/healthsites/facilities?page=2233&format=json`. So, the total number of facilities will be something close to 223 300 (maybe the last page, do not contain 100 healtsites).

To find the value `2233` of the `page parameter` I was obliged to make successive call to the url, incrementing the `page parameter` for each call, until an empty list was returned. 

It worked, but I made useless calls to the API, it's time and resources consuming. The best way in my humble opionion, is to return healthsites by chunk of 100, but with each chunk, return a `total field` indicating what is actually the total number of healthsites available in the database.

The function returning all  the healthsites seems to be:
```
def get_heathsites_master():
    return Locality.objects.filter(is_master=True).order_by('name')
```

One idea could be: everywhere we return healthsites, also return the total: `total = Locality.objects.filter(is_master=True).order_by('name').count()`
